pkg/daemon/admission/tlsutil.go

```diff
@@ -0,0 +1,80 @@
+/*
+ Copyright (c) 2019 Multus Authors
```

Comment:
Why is it Multus Authors?

Reply:
Original file belongs to multus team. Hence their header